src/components/Layouts/_index.scss

```diff
@@ -0,0 +1,104 @@
+////
+/// Modular layouts styles.
+/// @group modular-layouts
+/// @copyright IBM Security 2020
+////
+
+@import '@carbon/colors/scss/colors';
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/theme-maps';
+
+/// Generate a map of spacing tokens from the provided Carbon theme map.
+/// @access private
+/// @param {Map} $theme-map - Map of theme variables.
+/// @returns {Map} - The filtered map of only spacing variables.
+@function get-spacing-tokens-map($theme-map) {
+  $-map: ();
+
+  @each $token, $value in $theme-map {
+    @if type-of($value) == 'number' {
+      $-map: map-merge(
+        $-map,
+        (
+          $token: $value,
+        )
+      );
+    }
+  }
+
+  @return $-map;
+}
+
+/// Map of spacing tokens.
+/// @type Map<String, Number>
+/// @access private
+$spacing-tokens-map: get-spacing-tokens-map($carbon--theme--g100);
+
+// Generate styles for the debugging message.
+@mixin debug {
+  @include carbon--type-style($name: caption-01);
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: $spacing-02;
+  left: 0;
+  color: $carbon--black-100;
+  background-color: $carbon--magenta-40;
+}
+
+// Generate "top" padding classes and styles for each spacing token.
+@mixin spacing-top {
+  @each $token, $value in $spacing-tokens-map {
+    .spacing--#{$token}--top {
+      padding-top: $value;
+      position: relative;
```

Comment:
Would you mind clarifying the purpose of the `position` property here? If it's to assist with debugging, I recommend including it as part of the `debug-spacing-*` mixins to ensure it doesn't introduce any unintended side effects.

Reply:
Relative positioning needs to be applied so that the absolutely-positioned pseudo elements can appear in the correct spot.

This is related to debugging though so I can move it there. 👍